config/environment.rb

```diff
@@ -13,3 +13,4 @@
   authentication: :plain,
   enable_starttls_auto: true,
 }
+Rails.logger = Logger.new(STDOUT)
```

Comment:
I wonder whether instead of adding this line, the `RAILS_LOG_TO_STDOUT` environment variable should be set in production:

https://github.com/getguesstimate/guesstimate-server/blob/bc021f49529f0c0427f20a6db49303e3a5fac67c/config/environments/production.rb#L29-L33

I ask because the addition of this line makes the output of `bundle exec rspec` extremely noisy.

Reply:
@quinn-dougherty any take here?

Reply:
not really. I think the terminal gets pretty noisy when we run these projects, but I don't really care.